csharp/Samples/Microsoft.Spark.CSharp/DStreamSamples.cs

```diff
@@ -69,7 +69,7 @@ internal static void DStreamTextFileSample()
                 () =>
                 {
 
-                    StreamingContext context = new StreamingContext(sc, 2);
+                    StreamingContext context = new StreamingContext(sc, 2000);
```

Comment:
consider switching to using the batchInterval variable below or adding comments on the unit of the second param in this sample and others below